### PR TITLE
Update gsoc2020ideas.md

### PR DESCRIPTION
**Pull request by adeyosemanputra**
add OWASP Risk Assessment Framework to gsoc 2020

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2020ideas.md [hunk 1]
```diff
@@ -221,3 +221,42 @@ Have a look at our GitHub [wiki page](https://github.com/blabla1337/skf-flask),
 * [Glenn ten Cate](mailto:glenn.ten.cate@owasp.org)
 * [Riccardo ten Cate](mailto:riccardo.ten.cate@owasp.org)
 
+### [OWASP Risk Assessment Framework](https://github.com/OWASP/RiskAssessmentFramework)
+
+##### Explanation of Ideas
+
+The OWASP Risk Assessment Framework consist of Static application security testing and Risk Assessment tools, Eventhough there are many SAST tools available for testers, but the compatibility and the Environement setup process is complex. By using OWASP Risk Assessment Framework's Static Appilication Security Testing tool Testers will be able to analyse and review their code quality and vulnerabilities without any additional setup. OWASP Risk Assessment Framework can be integrated in the DevSecOps toolchain to help developers to write and produce secure code.
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2020ideas.md]: > tools, Eventhough

Should there be a period there, or should "eventhough" start lower case?

File: pages/initiatives/gsoc/gsoc2020ideas.md [hunk 2]
```diff
@@ -221,3 +221,42 @@ Have a look at our GitHub [wiki page](https://github.com/blabla1337/skf-flask),
...
+* Add another tool other than sonarqube (Integrate other tools to Raf)
+* Make some generate report into pdf, excel, word etc
+* put the OWASP RAF into cloud so it's accessible
+* Make our tool support few other languages as well
+* work in docker so our tools can install by docker
+
+#### 1st Idea
+* add other tools features like IAST, RASP etc
+
+#### 2st Idea
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2020ideas.md]: 2nd

File: pages/initiatives/gsoc/gsoc2020ideas.md [hunk 3]
```diff
@@ -221,3 +221,42 @@ Have a look at our GitHub [wiki page](https://github.com/blabla1337/skf-flask),
...
+* please see my Expected Results 
+
+#### 3st Idea
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2020ideas.md]: 3rd

File: pages/initiatives/gsoc/gsoc2020ideas.md [hunk 4]
```diff
@@ -221,3 +221,42 @@ Have a look at our GitHub [wiki page](https://github.com/blabla1337/skf-flask),
...
 
+### [OWASP Risk Assessment Framework](https://github.com/OWASP/RiskAssessmentFramework)
+
+##### Explanation of Ideas
+
+The OWASP Risk Assessment Framework consist of Static application security testing and Risk Assessment tools, Eventhough there are many SAST tools available for testers, but the compatibility and the Environement setup process is complex. By using OWASP Risk Assessment Framework's Static Appilication Security Testing tool Testers will be able to analyse and review their code quality and vulnerabilities without any additional setup. OWASP Risk Assessment Framework can be integrated in the DevSecOps toolchain to help developers to write and produce secure code.
+
+##### Expected Results
+* A new features with awesome 
+* Make some improvement in scanner (Sast tool)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2020ideas.md]: Should SAST be full caps? (As an acronym)

File: pages/initiatives/gsoc/gsoc2020ideas.md [hunk 5]
```diff
@@ -221,3 +221,42 @@ Have a look at our GitHub [wiki page](https://github.com/blabla1337/skf-flask),
...
+* Add another tool other than sonarqube (Integrate other tools to Raf)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2020ideas.md]: Should RAF be full caps? (As an acronym)

File: pages/initiatives/gsoc/gsoc2020ideas.md [hunk 6]
```diff
@@ -221,3 +221,42 @@ Have a look at our GitHub [wiki page](https://github.com/blabla1337/skf-flask),
...
+* Angular 7/8
+* Typescript
+* Web Development Skills
+* Docker
+* Java
+* Mongodb
+
+##### Getting started:
+* good knowledge about secure coding and other tools https://owasp.org/www-community/Source_Code_Analysis_Tools
+* contact mentor for discuss your ideas ( ade.putra@owasp.org)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2020ideas.md]: mailto? or remove the space after the bracket.

**Comment by hblankenship** [comment]
Please resolve the items listed.

**Comment by kingthorin** [comment]
Also needs to be de-conflicted.

File: pages/initiatives/gsoc/gsoc2020ideas.md (hunk 6 above)
Comment (adeyosemanputra) [reply pages/initiatives/gsoc/gsoc2020ideas.md]: okay

**Comment by adeyosemanputra** [comment]
sorry we are different time zone, i have fix some items.please review again

File: pages/initiatives/gsoc/gsoc2020ideas.md (hunk 5 above)
Comment (adeyosemanputra) [reply pages/initiatives/gsoc/gsoc2020ideas.md]: fix it

**Comment by kingthorin** [comment]
Do you need help addressing the conflict?

**Comment by adeyosemanputra** [comment]
> Do you need help addressing the conflict?

yes can you help me for resolve the conflict

**Comment by kingthorin** [comment]
Sure:
1. You need to fetch the latest from the OWASP/www-community repo. (Normally `git fetch upstream`)
2. Then you need to merge `upstream/master` into your branch.
3. It'll complain and you'll need to address the conflicts in `gsoc2020ideas.md`.
4. Address the conflicts (make sure you have all the latest content plus your new stuff). [I see that you did try to add the SAMM stuff but that isn't the latest, it's missing the PCSI > PCI fix.]
5. Add the file back `git add .` or `git add \path\to\gsoc2020ideas.md`
6. Commit and push again.

(Since you're working on master not a feature branch in your fork/repo you might have to do a forced push to make it work.)

**Comment by adeyosemanputra** [comment]
> Sure:
> 
> 1. You need to fetch the latest from the OWASP/www-community repo. (Normally `git fetch upstream`)
> 2. Then you need to merge `upstream/master` into your branch.
> 3. It'll complain and you'll need to address the conflicts in `gsoc2020ideas.md`.
> 4. Address the conflicts (make sure you have all the latest content plus your new stuff). [I see that you did try to add the SAMM stuff but that isn't the latest, it's missing the PCSI > PCI fix.]
> 5. Add the file back `git add .` or `git add \path\to\gsoc2020ideas.md`
> 6. Commit and push again.
> 
> (Since you're working on master not a feature branch in your fork/repo you might have to do a forced push to make it work.)

ok, will close this pull request and create other pull request to avoid conflict (easy way)

**Pull request closed without merging**